crates/e2e/src/setup/solver/solution.rs

```diff
@@ -45,47 +45,58 @@ impl JitOrder {
         signing_scheme: EcdsaSigningScheme,
         domain: &DomainSeparator,
         key: SecretKeyRef,
-    ) -> solvers_dto::solution::JitOrder {
+    ) -> (solvers_dto::solution::JitOrder, OrderUid) {
```

Comment:
nanonit: uid is not dependent on signing and should (it's a pure coincidence that this method is called in the test where it is used, and happens to also depend on `DomainSeperator`). It makes more sense to have this as a standalone method imo.